forms-flow-web/src/_tests_/test-components/test-form/test-item/view.test.js

```diff
@@ -89,6 +90,29 @@ it("should render the public View component without breaking ", async () => {
   expect(applicationCreate).toHaveBeenCalled();
 });
 
+it("should call the custom submission when custom submission is on ",() =>{
+  constants.CUSTOM_SUBMISSION_ENABLE = true;
+  constants.CUSTOM_SUBMISSION_URL = true;
+  const spy = jest.spyOn(redux, "useSelector");
+  spy.mockImplementation((callback) =>
+    callback({
+      applications: { isPublicStatusLoading: false },
+      form: { isActive: false },
+      formDelete: { isFormSubmissionLoading: false },
+      user: { lang: "" },
+      draft: { draftSubmission: {} },
+    })
+  );
+  const customSubmission = jest.fn();
+  customSubmission(postCustomSubmission);
+  renderWithRouterMatch(View, {
+    path: "/public/form/:formId",
+    route: "/public/form/123",
+  });
+  fireEvent.click(screen.getByText("Submit"));
+  expect(customSubmission).toHaveBeenCalled();
```

Comment:
need to check the calling parameters too add one more test with `toHaveBeenCalledWith`
this test might be incorrect since you are calling the function in line 107.
I suggest using module mock and function mock together. use spyOn to mock the module and jest.fn() to mock the function, refer line 128 to 130